dix/req_internal.h

```diff
@@ -130,10 +130,10 @@ struct m0_dix_pg_unit {
 	enum m0_pool_nd_state  dpu_pd_state;
 
 	/**
-	 * Indicates whether this parity group is unavailable, e.g. the target
-	 * for the unit has failed and not repaired yet.
+	 * Indicates whether this parity unit is unavailable, e.g. the target
```

Comment:
Is this specific to parity unit only?

Reply:
No, wanted to fix the comment (should be a unit, not a group), but missed "parity" word, will fix.

Reply:
Suggestion: "parity group unit".

Reply:
Done.